test/index.js

```diff
@@ -103,7 +112,7 @@ test('validate-npm-package-name', function (t) {
   t.deepEqual(validate('CAPITAL-LETTERS'), {
     validForNewPackages: false,
     validForOldPackages: true,
-    warnings: ['name can no longer contain capital letters']})
+    warnings: ['name cannot start with a special/capital character', 'name can no longer contain capital letters']})
```

Comment:
Previous message seems to be more appropriate here.

Reply:
So, what happens is each package name gets checked against all the rules created in the `index.js`, and errors and warnings got pushed in the list so while writing a test case, we have to mention all the warnings. That' why there are 2 warnings here.